app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -4,23 +4,60 @@
 
 package org.mozilla.focus.browser
 
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
 import android.content.Context
+import android.graphics.Color
 import android.graphics.drawable.Drawable
 import android.support.v4.content.ContextCompat
+import android.support.v7.widget.GridLayoutManager
+import android.support.v7.widget.RecyclerView
 import android.util.AttributeSet
+import android.util.TypedValue
 import android.view.LayoutInflater
 import android.view.View
+import android.view.ViewGroup
+import android.view.animation.DecelerateInterpolator
 import android.widget.LinearLayout
 import kotlinx.android.synthetic.main.browser_overlay.view.*
+import kotlinx.android.synthetic.main.home_tile.view.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.